bolt-simple-taproot.md

```diff
@@ -0,0 +1,1141 @@
+combine the public nonces into the aggregate nonce which will be a part of the
+final signature.
+
+The `Sign` method is used to generate a valid partial signature, with the
+`PartialSigVerify` algorithm used to verify each signature.
+
+Once all partial signatures are known, the `PartialSigAgg` algorithm is used to
+aggregate the partial signature into a final, valid [BIP
+340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki) Schnorr
+signature.
+
+##### Partial Signature Encoding
+
+A standard `musig2` partial signature is simply the encoding of the `s` value,
+which is a 32-byte-element modulo the order of the group.
+
+Throughout this specification, we will be using a custom type,
+`PartialSignatureWithNonce`, comprised of the aforementioned `s` value, along
+with the 66-byte-encoding of the public nonce used to sign remote commitments:
+
+```
+s || point_1 || point_2
+```
+
+## Design Overview
+
+With the preliminaries out of the way, we provide a brief overview of the
+Simple Taproot Channel design.
+
+The multisig output becomes a single P2TR key, with `musig2` key aggregation
+used to combine two keys into one.
+
+For commitment transactions, we inherit the anchor output semantics, meaning
+there are two outputs used for CPFP, with all other scripts inheriting a `1
+CSV` restriction.
+
+The local output of the commitment transaction uses the revocation key as the
+taproot internal key, and commits to a single script path of the delay script.
+
+The remote output of the commitment transaction uses the `combined_funding_key`
+as the top-level internal key, and then commits to a normal remote script.
+
+Anchor outputs use the respective two parties' funding keys as the top-level
+internal key committing to the script `16 CSV`. If a local or remote output is
+missing from the commitment, its corresponding anchor must not be present,
+either.
+
+All HTLC scripts use the revocation key as the top-level internal key. This
+allows the revocation to be executed without additional on-chain bytes, and
+also reduces the amount of state that nodes need to keep in order to properly
+handle channel breaches.
+
+## Specification
+
+### Feature Bits
+
+Inheriting the structure put forth in BOLT 9, we define a new feature bit to be
+placed in the `init` message, `node_announcement` and also
+`channel_announcement`. We opt to also place the bit in the
+`channel_announcement` so taproot channels can be identified on the routing
+level, which will eventually matter once PTLCs are fully rolled out.
+
+
+| Bits  | Name             | Description                           | Context | Dependencies                             | Link                 |
+|-------|------------------|---------------------------------------|---------|------------------------------------------|----------------------|
+| 30/31 | `option_taproot` | Node supports simple taproot channels | IN      | `option_channel_type` + `option_anchors` | TODO(roasbeef): link |
```

Comment:
can say `option_anchors_zero_fee_htlc_tx` as the other one is unsafe (even if more verbose though better than an ill-implemented channel negotiation logic setting legacy option_anchors for simple taproot channels :) )

Reply:
IIRC, we made a change to just always refer to the final anchor as simpluy `option_anchors`.